run_create_eb_windows.py

```diff
@@ -116,9 +120,10 @@ def run_create_eb_windows(name, settings):
     subprocess.Popen(['mkdir', '-p', template_path]).communicate()
 
     if phase == 'dv':
-        git_command = ['git', 'clone', '--depth=1', git_url]
+        git_command = ['git', 'clone', '--depth=1', '-b', sub_branch, git_url]
```

Comment:
@singetta // DV에서만 브랜치를 선택할수 있게 해놨기 때문에 어떤 브랜치를 땡겨오는지 직전에 알려주면 좋을것 같습니다.
```suggestion
        print(f'dv branch: {dv_branch}')
        git_command = ['git', 'clone', '--depth=1', '-b', dv_branch, git_url]
```